src/appleseed-max-impl/appleseedsssmtl/appleseedsssmtl.cpp

```diff
@@ -828,3 +834,12 @@ HINSTANCE AppleseedSSSMtlClassDesc::HInstance()
 {
     return g_module;
 }
+
+bool AppleseedSSSMtlClassDesc::IsCompatibleWithRenderer(ClassDesc& rendererClassDesc)
+{
+    Class_ID classID = rendererClassDesc.ClassID();
```

Comment:
The whole method could be simplified to

    return renderer_class_desc.ClassID() == AppleseedRenderer::get_class_id();

This applies to other materials as well.